### DISCUSSION TITLE
Resize shortuct is not working. How do I troubleshoot?

### DISCUSSION THREAD

**Discussion by jennydaman** [lingtjien/Grid-Tiling-Kwin#104] [Q&A]
Every other shortcut works (such as `Grid-tiling: Tile/Float` and `Grid-Tiling: Move/Swap Left`). I am inexperienced with Kwin script debugging. How can I do troubleshooting? In the Kwin script's settings, "Step Size" is `0.05`. I think my settings are correct, though one can never be sure.

**Comment by lingtjien** [comment]
You have to focus a client (window) and then press the shortcut and then it will increase (or decrease) the size of that client.

If that doesn't works then check the global shortcut section KWin and search for `Grid-Tiling` to check if the shortcut has been registered

**Reply by jennydaman** [reply]
I have been a long time user of `Grid-Tiling-Kwin` since 2020-07-02. It was only recently that I've realized, oh wait, I haven't updated this ever... After `git pull` to latest master, the resize shortcuts stopped working. I've tried checking out through `HEAD^^` but it didn't work. Maybe I should go further back?

**Reply by jennydaman** [reply]
Also tried uninstalling and reinstalling from scratch, of course. Did not work.

**Comment by jennydaman** [comment]
It works when I checkout db4a64a

I will do a binary search from then to current `HEAD` and find out why it does not work.

**Comment by jennydaman** [comment]
The latest working commit for me is 4083203f85f117dc0ede45cc4ffddf9f3dc7202f, which was tagged [0.0.1](https://github.com/lingtjien/Grid-Tiling-Kwin/releases/tag/0.0.1)

My two laptops both run Arch Linux, KDE Plasma version 5.21.4, KDE Frameworks version 5.80.0

On master branch, if I press CTRL++, in some windows (like systemsettings) it starts typing `+` sign. But in `kitty` nothing happens.

**Comment by lingtjien** [comment]
I assume you're on X11?

Can you go to the global shortcut settings and then the KWin section. Then type in the search bar `Grid-Tiling` and make a screenshot of that

**Reply by jennydaman** [reply]
I am on X11.

![keyboard_shortcuts](https://user-images.githubusercontent.com/20404439/114104622-374acf80-9899-11eb-9a56-35fc3638ff7a.png)

![script_settings](https://user-images.githubusercontent.com/20404439/114104776-87299680-9899-11eb-8832-61525fbf2f5e.png)

**Comment by lingtjien** [comment]
Alright, so the problem is that some of the shortcuts were renamed from previous versions.

As you can see in the shortcut list you have `Meta++` assigned to `Grid-Tiling: Increase Size` but that is the old shortcut name, the new name is now `Grid-Tiling: Increase Step` and `Grid-Tiling: Increase Max`.

You should remove the old shortcuts (since they will only confuse you) and then assing the correct shortcuts to the new ones. Removing the old shortcuts can be done as described in the *Useful to know* section of the readme

> If you have KWin shortcuts added by the script that you no longer need or use (because they got renamed to something else for example), then you can delete the old unused shortcuts by first selecting KWin in the global shortcuts menu and then pressing the remove the selected component button (hover mouse over the button to see the text). This action does require you to not have the KWin script active.

**Reply by jennydaman** [reply]
That makes sense. It worked for me, thank you so much.
